doc/source/composites.rst

```diff
@@ -536,6 +536,23 @@ the file) as::
           kwargs:
             gamma: [1.7, 1.7, 1.7]
 
+.. warning::
+   If you define a composite with no matching enhancement, Satpy will by
+   default apply a crude stretch enhancement.  If you want no enhancement
```

Comment:
I think technically it isn't a crude stretch but the "linear" stretch which defaults to a 5% cutoff/quantile on either end.

https://github.com/pytroll/trollimage/blob/93e547a45a1c3a13768da4a5169a3f5bf5ebc75a/trollimage/xrimage.py#L1014

Reply:
Fixed in 8dba3d9.